nightly/Dockerfile

```diff
@@ -19,24 +19,27 @@ ENV DB_SERVER localhost
 ENV DB_USER macfly
 ENV PS_ENABLE_SSL 1
 
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt update \
+  && apt install -y default-mysql-server ssl-cert \
+  && rm -rf /var/lib/apt/lists/*
+
+# XDebug extension
+RUN pecl install xdebug \
+    && docker-php-ext-enable xdebug
```

Comment:
These images are needed to reproduce production environments as well. xDebug may have an unwanted impact on them, so it seems appropriate to condition its installation or at least its enablement.